routers/web/repo/pull_review.go

```diff
@@ -242,3 +245,43 @@ func DismissReview(ctx *context.Context) {
 
 	ctx.Redirect(fmt.Sprintf("%s/pulls/%d#%s", ctx.Repo.RepoLink, comm.Issue.Index, comm.HashTag()))
 }
+
+const headCommitKey = "_headCommitSHA"
+
+func UpdateViewedFiles(ctx *context.Context) {
+	pull := checkPullInfo(ctx).PullRequest
```

Comment:
NPE `checkPullInfo` can return `nil`

Reply:
This isn't fixed `checkPullInfo(ctx)` can return `nil` doing a property access on `nil` would still lead to NPE

Reply:
Ah, I misunderstood your comment.
Yeah, valid point.